db/initialize.js

```diff
@@ -20,7 +20,7 @@ const initializeDatabase = async () => {
     alterandforce = false;
     // Sync models with database
     await sequelize.sync({
-      alter: false,
+      alter: true,
```

Comment:
Let's always keep this `false` in the repository.
To make changes to the database, SSH into the instance, set `alter: true`, restart the server, then set `alter:false`, and restart the server again. 

Committing `alter: true` to the repository risks that future changes will all contain `alter: true` and the database will needlessly be updated with every change.